packages/browser/__tests__/login/oidc/redirectHandler/AuthCodeRedirectHandler.spec.ts

```diff
@@ -292,12 +325,17 @@ describe("AuthCodeRedirectHandler", () => {
     });
 
     it("returns an authenticated dpop fetch if requested", async () => {
-      window.fetch = jest.fn().mockReturnValueOnce(
+      window.fetch = jest.fn() as jest.Mock<
+        ReturnType<typeof window.fetch>,
+        [RequestInfo, RequestInit?]
+      >;
+      window.fetch = jest.fn().mockReturnValue(
```

Comment:
Setting `window.fetch` here, and then again to something else immediately afterwards - I assume this is not intentional?

Reply:
8bf476ae96ffd631fd3e9db7b10cf696d314c786